Neos.Kickstarter/Classes/Command/KickstartCommandController.php

```diff
@@ -47,7 +47,6 @@ class KickstartCommandController extends CommandController
      *
      * @param string $packageKey The package key, for example "MyCompany.MyPackageName"
      * @param string $packageType Optional package type, e.g. "neos-plugin"
-     * @return string
```

Comment:
Strictly speaking (as per our CGL) `@return` must be given, unless it's on `__construct` or a return type declaration is present. So why not add the latter (for all those places)?

Reply:
hmm yes true, thats on my board for the later phpstan levels where it will also be enforced okay?